org.gnome.Radio.json

```diff
@@ -0,0 +1,105 @@
+{
+  "app-id" : "org.gnome.Radio",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "43",
+  "sdk" : "org.gnome.Sdk",
+  "command" : "gnome-radio",
+  "finish-args" : [
+     "--socket=x11",
+     "--share=ipc"
```

Comment:
you enable wayland everywhere but don't allow it here. Also DRI is probably needed too. And I don't see something for the sound.